WordPress/WordPressTest/WPAccount+LookupTests.swift

```diff
@@ -1,14 +1,7 @@
 import XCTest
 @testable import WordPress
 
-final class WPAccountLookupTests: XCTestCase {
-
-    private var contextManager: ContextManagerMock!
-
-    override func setUp() {
-        super.setUp()
-        contextManager = ContextManagerMock()
-    }
+final class WPAccountLookupTests: CoreDataTestCase {
```

Comment:
Might as well remove the `final` while we're at it:

```suggestion
class WPAccountLookupTests: CoreDataTestCase {
```

I don't have the details at hand, but if I recall correctly, there was a time in which marking test cases as `final` helped with compilation speed. Unless I'm mistaken, that's no longer the case, so these can be avoided.